vm-setup/roles/v1aX_integration_test/filter_plugins/status.py

```diff
@@ -0,0 +1,63 @@
+""" filtering k8s_info resources """
+
+from ansible.errors import AnsibleFilterError
+from ansible.utils.display import Display
+
+display = Display()
+
+
+def filter_running(a, skip_missing=False):
```

Comment:
I think the default behaviour should be to warn, rather than error. If there's no `["status"]["phase"]` key, I think we can safely assume that whatever the resource is, it's not in the running state that we want, so let's warn + print the details about the resource, and continue, not appending the resource to the list.

Would it also be possible to add this behaviour to the other filters you've written? I believe all of them could encounter the same KeyError.